playhand.js

```diff
@@ -532,7 +555,26 @@ function myPokerHand(handCards, boardCards) {
     });
 
     // Check straight flush
-    // will do this later
+    var flush = false
+    var straightFlush = false
+    Object.entries(suitToCount).forEach(([suit, count]) => {
+        if (count >= 5) {
+            flush = true
+            const cardsOfSuit = []
+            allCards.forEach(card => {
+                if (card.suit === suit) {
+                    cardsOfSuit.push(card)
+                }
+            })
```

Comment:
should be able to do something like this
```suggestion
            const cardsOfSuit = allCards.filter(card => card.suit === suit)
```